src/Microsoft.CSharp/pkg/Microsoft.CSharp.pkgproj

```diff
@@ -1,10 +1,7 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
-    <ProjectReference Include="..\ref\Microsoft.CSharp.csproj">
```

Comment:
The PR removes any redundant ref projects (those that are no different than the ones we'd harvest).  I'm keeping the ref projects around but haven't yet built infrastructure to do filtering.  As we mentioned filtering is non-trivial since things like versions change.  I'll need to build some assembly diffing in order to properly filter.